src/Azure.Functions.Cli/Common/AuthSettingsFile.cs

```diff
@@ -0,0 +1,81 @@
+﻿using Azure.Functions.Cli.Common;
+using Colors.Net;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+namespace Azure.Functions.Cli.Common
+{
+    class AuthSettingsFile
+    {
+        public bool IsEncrypted { get; set; }
+        public Dictionary<string, string> Values { get; set; }
+        private readonly string _filePath;
+        private const string reason = "secrets.manager.auth";
+
+        public AuthSettingsFile(string filePath)
+        {
+            _filePath = filePath;
+            try
+            {
+                string path = Path.Combine(Environment.CurrentDirectory, _filePath);
+                var content = FileSystemHelpers.ReadAllTextFromFile(path);
+                var authSettings = JObject.Parse(content);
+                Values = new Dictionary<string, string>(authSettings.ToObject<Dictionary<string, string>>(), StringComparer.OrdinalIgnoreCase);
+            }
+            catch
+            {
+                Values = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
+                IsEncrypted = false;
+            }
```

Comment:
If something goes wrong, we need to provide some kind of signal so that users can either fix the issue themselves or report the details to us. For that reason, I suggest the following:
1. Check to see if the file exists before trying to load it (`File.Exists(filePath)`). If it doesn't exist, we can silently move on like you're currently doing.
2. If there is a JSON parsing exception, log the message of the exception so that the user can figure out why their JSON file was invalid.
3. If it's a file permission error, log just the exception message.
4. Any other exception type, log the full exception details.